Description: fix test for nodejs >= 16
Author: Yadd <yadd@debian.org>
Forwarded: no
Last-Update: 2022-05-04

--- a/test/index.js
+++ b/test/index.js
@@ -245,9 +245,7 @@
         checkStream.on('error', function (err) {
           assert.ok(err)
           erred = true
-        })
-        writeStream.on('close', function () {
-          if (erred) return done()
+          done()
         })
       })
     })
@@ -273,9 +271,7 @@
         checkStream.on('error', function (err) {
           assert.ok(err)
           erred = true
-        })
-        writeStream.on('close', function () {
-          if (erred) return done()
+          done();
         })
       })
     })
